### PR TITLE
Adding support for Ubuntu 22.04 Jammy

### LINKED ISSUE #31: FYI: No longer works on Ubuntu 22.04 (Daily Build)
Just a heads up; and realizing this is very early Beta.

### PR DESCRIPTION
**Pull request by mendhak**
Hello again.  Just like Impish, in Jammy (Ubuntu 22.04)  the wallpaper is being set in `gdm.css` instead of `gdm3.css`. 

Fixes #31 and also includes the text 'impish' for @duckimann peace of mind: https://github.com/thiggy01/change-gdm-background/pull/29#issuecomment-962546848 

I tested this in a VM using the [daily build image](https://cdimage.ubuntu.com/daily-live/current/).  

VirtualBox_Ubuntu2204_14_11_2021_21_38_54

VirtualBox_Ubuntu2204_14_11_2021_21_39_04

### CODE REVIEW

**Comment by mendhak** [comment]
PS I've been using your script with Variety wallpaper changer for a while now and it works really well.  I'm now able to keep the desktop and login screen wallpapers synchronized.  I also did a small [writeup](https://code.mendhak.com/synchronize-login-wallpaper-ubuntu/) on how to use them together, and also how to use this script in a cron job.  Many thanks.

**Comment by XaTTa6bl4** [comment]
@thiggy01 Please, merge this. Ubuntu 22.04 already in production.

**Comment by duckimann** [comment]
> @thiggy01 Please, merge this. Ubuntu 22.04 already in production.

TLDR; Doesn't check the gresource file version, if the script f-up somewhere, you'll have to use an ISO to fix the problem.

If i remember correctly, this commit still have problem with the backup of the gresource file. If you used this script in Ubuntu 20.04 then upgrade it to newest version then the script still think that it's the backup of the current version and if it messed up the gresource file, i bet you'll have to find the gresource file in an ISO file.

**Comment by arindam-giri** [comment]
Hi All. You should make the changes ONLY FOR JAMMY, as I explained here. And tested as well. 
https://www.debugpoint.com/change-login-background-ubuntu/

Can anyone make a pull req with the above two changes and add an "IF-ELSE" block to switch between gdm3.css and gdm.css before Jammy and after Jammy onwards?. I believe it should work. Cheers.

**Comment by mendhak** [comment]
In Ubuntu 22.04/24.04 there is a safer way of setting the background with gsettings and machinectl, and doesn't involve modifying the CSS files, so I'm closing this PR. 

I have a [blog post here](https://code.mendhak.com/synchronize-login-wallpaper-ubuntu/), and a repo with [sample scripts here](https://github.com/mendhak/ubuntu-change-login-background).
